<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choose_auto_calculation_type_dialog</class>
 <widget class="QDialog" name="choose_auto_calculation_type_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>155</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Выберите действие</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="main_widget_layout" stretch="0">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выберите тип добавления коэффициентов.&lt;/p&gt;&lt;p&gt;При выборе &lt;span style=&quot; font-weight:600;&quot;&gt;Дополнить&lt;/span&gt; рассчитанные коэффициенты будут добавлены к уже существующим.&lt;/p&gt;&lt;p&gt;При выборе &lt;span style=&quot; font-weight:600;&quot;&gt;Заменить&lt;/span&gt; существующие коэффициенты будут стерты.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttons_layout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="add_button">
       <property name="text">
        <string>Дополнить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replace_button">
       <property name="text">
        <string>Заменить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
